hooks.mako

```diff
@@ -1,7 +1,18 @@
 ## -*- coding: utf-8 -*-
 <%inherit file="base.mako" />
 
-<div class="page-header"><h1>Supported hooks</h1></div>
+<div class="page-header">
+    <h1>Supported hooks</h1>
+    <div class="filter">
+        <input id="search-hook-id" type="text" placeholder="filter hook id"></input>
```

Comment:
for the smallest responsive size, this needs more margin

I wonder if there's a bootstrap forms thing that does all this for you

Reply:
i restructured the header with bootstrap `col` and `row`.